### DISCUSSION TITLE
[dropped] Настройка hotkey

### DISCUSSION THREAD

**Discussion by Qb1cc** [totumonline/totum-ru-issues-and-discussions#141] [Есть идея, а что если...]
Тотуму необходима возможность натройки hotkey , для более быстрого взаимодействия. Мне достаточно привычно использовать табы и всё что есть уже в тотум, но многим пользователям не возможно донести что то новое, проще создать условия под них, поэтому необходимо сделать привязку кнопок клавиатуры к действиям внутри тотум.

Как пример, я нажимаю C и создаётся новый цикл, ( Create ) и т.п

**Comment by ttm-alexeypolunin** [comment]
Принято, но не все можно переназначить тк для браузеров есть жестко забинденные комбинации.
